webapp/templates/_footer.html

```diff
@@ -0,0 +1,6 @@
+<footer>
+  <section class="caveat">
+    This information is intended for web platform implementers and does not contain
+    useful metrics for evaluation or comparison of web platform features.
```

Comment:
How about:

"The Web Platform Tests Dashboard is a work-in-progress. The reported results do not necessarily reflect the true capabilities of each web browser."

Reply:
Oh, err, I think maybe the text I'm recommending belongs in the header.

Reply:
I've updated the top banner :)